modules/gitops-release-notes-1-3-2.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3-2_{context}"]
+= Release notes for {gitops-title} 1.3.2
+
+{gitops-title} 1.3.1 is now available on {product-title} 4.6, 4.7, 4.8, and 4.9.
+
+[id="fixed-issues-1-3-2_{context}"]
+== Fixed issues
+
+* Previously, in the `OperaturHub` if you filter by "Disconnected" under the *Infrastructure Features* the {gitops-title} did not appear, since it the Operator did not had the related annotation set in its CSV. The {gitops-title} is now annotated with "Disconnected Cluster" as an *Infrastructure Feature*.
+
+* When using an "Application-scoped" argocd (e.g. one that is not scoped to All Namepsaces in a cluster), the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces includes the `argocd.argoproj.io/managed-by` label. This list of namespaces gets cached in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. However, if one of these namespaces gets deleted, the operator doesn't seem to notice, and the namespace remains in that list. This breaks both the *CONNECTION STATE* in that cluster config, and also all sync attempts after the namespace gets deleted. The sync will show errors. For example:
```

Comment:
I am now confused, do we call it `application-scoped` Argo CD or `namespace scoped` Argo CD. Can you tell me where did you find the phrase `application-scoped` Argo CD ?